### DISCUSSION TITLE
[Feature Request] Share with: "People"

### DISCUSSION THREAD

**Discussion by CarolinaCst** [owncloud/web#5291] [Ideas]
### Scope: Phoenix ownCloud

Actual Landing Page: 
https://web.owncloud.com/

### Steps to reproduce
1. Open homepage
2. Click on "Share with others"
3. The share documents are other in two options: "Share with" and "Share time"

### Expected behaviour

- Users can only see the documents

Tell us what should happen

### Actual behaviour
Tell us what happens instead

**Comment by fschade** [comment]
@CarolinaCst, thank you for your feature request, can you explain a bit further what you mean by that `The share documents are other in two options: "Share with" and "Share time"`
